charts/argo-cd/templates/argocd-application-controller/clusterrole.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.controller.name "name" .Values.controller.name) | nindent 4 }}
 rules:
+{{- if .Values.controller.clusterRoleRules.enabled }}
+{{- toYaml .Values.controller.clusterRoleRules.rules | nindent 2 }}
```

Comment:
Hi @loreleimccollum-work,
Have you tried the new release with success?
I think this should be `nindent 0` instead of `2`

Reply:
yeah we are actually currently using it

Reply:
I think on our side when we pass it in, we are passing in a template yaml file like 
`- apiGroups:
  - ""
  resources:
  - configmaps
  - endpoints
  - persistentvolumeclaims
  - persistentvolumeclaims/status
  - pods
  - replicationcontrollers
  - replicationcontrollers/scale
  - serviceaccounts
  - persistentvolumes
  - podtemplates
  - bindings
  - events
  - ingresses
  - limitranges
  - namespaces/status
  - pods/log
  - pods/status
  - replicationcontrollers/status
  - resourcequotas
  - resourcequotas/status
  - namespaces
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - networking.k8s.io
  - extensions
  - discovery.k8s.io
  - cert-manager.io
  - batch
  - gatekeeper.sh
  - externalsecrets.kubernetes-client.io
  - apiextensions.k8s.io
  - apiregistration.k8s.io
  - acme.cert-manager.io
  resources:
  - '*'
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - traefik.containo.us
  - argoproj.io
  - rbac.authorization.k8s.io
  - apps
  resources:
  - '*'
  verbs:
  - '*'
- apiGroups:
  - ""
  resources:
  - services
  - workflows
  verbs:
  - get
  - list
  - watch
  - create
  - update
  - patch
  - delete
- nonResourceURLs:
  - '*'
  verbs:
  - get
  - list
  - watch
`
So it could have to do with how you pipe in the stuff?

Reply:
Ahh, I see. After this `toYaml | indent 2` follows an `else` statement. Then this works but the indentation is different as when using the defaults.